addons/theme_default/data/theme_default_data.xml

```diff
@@ -1,8 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
-<odoo>
-
-<record id="website.default_website" model="website">
-    <field name="theme_id" ref="base.module_theme_default"/>
-</record>
-
-</odoo>
```

Comment:
Should not we remove the file entirely?

Reply:
pour moi oui, meme le dossier data et models sont useless...

Reply:
Y'a plus de dossier model vu que plus de fichier dedans ;) Mais oui, retirons le fichier data donc ;)